restore_backup.sh

```diff
@@ -0,0 +1,29 @@
+BACKUP_FILE=$1
+ADMIN_PASS=$2
+
+if [[ -z "$BACKUP_FILE" ]] ; then
+    echo 'Please, provide backup file name as an argument.'
+else
```

Comment:
I would avoid to have if else in this case I would do :
if [[ -z "$BACKUP_FILE" ]] ; then
    echo 'Please, provide backup file name as an argument.'
    exit 1
fi
 if [ -z "$ADMIN_PASS" ] ; then
        ADMIN_PASS=geoserver
  fi